dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsOrderBookStreamTestSuite.scala

```diff
@@ -76,6 +77,40 @@ class WsOrderBookStreamTestSuite extends WsSuiteBase {
   }
 
   "Order book stream should" - {
+    "send prices in asks with right order" in {
+      val wsc = mkDexWsConnection(dex1)
+      wsc.send(WsOrderBookSubscribe(wavesUsdPair, 1))
+
+      (1 until 50).foreach(i => placeAndAwaitAtDex(mkOrderDP(alice, wavesUsdPair, BUY, 1.usd, i * 10000)))
```

Comment:
This code places orders one by one. This means, we will receive updates one by one too. 
Consider place all order (see the task description), then wait until all orders are placed. And then `receiveAtLeastNRaw(1)` - there could be more, but this is okay.